src/lib/EventPublisher.js

```diff
@@ -77,9 +77,9 @@ class EventPublisher extends EventEmitter {
 
 
   subscribe = (
-    eventName: ?string = ALL_EVENTS,
+    eventName: string = ALL_EVENTS,
```

Comment:
hm..if I do `eventName: ?string = ALL_EVENTS,` Flow doesn't understand that `eventName` is always not null and throws errors about it later on..


`eventName: string = ALL_EVENTS,` - this not correct also, because eventName could be `?string`


I can don't user default params and do 
`eventName: ?string`
and later:
`this.on(eventName || ALL_EVENTS, ...)` - but this doesn't look beautiful..
what is the best pattern for these kinds of situations?
the same with `filterOptions`